kubernetes/tests/metric_test.yaml

```diff
@@ -117,6 +117,9 @@ spec:
         dimensions:
           - bucket_name: sample-s3-bucket
             filter_id: EntireBucket
+      nginx:
+        dimensions:
+          - namespace: sample_nginx_namespace
```

Comment:
namespace might be a too big area. There might be multiple nginx's on different apps running in the same namespace.

Reply:
No no. This is not kubernetes namespace. This is nginx namespace. So, for example, one nginx namesapce's logs could be getting dumped in `/var/log/nginx/myapp1/access.log` and another nginx namespace's logs could be getting dumped in `/var/log/nginx/myapp2/access.log`.

Reply:
got it. for simplicity can we have the dimension name `nginx_namespace` instead of `namespace` then?